docs/programming/kotlin/fundamentals/functions.md

```diff
@@ -0,0 +1,11 @@
+# Functions
+
+## Declaring a Function
+
+### Using Functions
+
+### Arguments & Parameters
+
+## Return Types
+
```

Comment:
Lambdas could be something you could potentially introduce - they're used with the command based framework in WPI so yeah. I may have said lambdas were irrelevant earlier so oops.